app/playground/README.md

```diff
@@ -0,0 +1,4 @@
+
+# playground
+
+This service just serves a Jupyter notebook in an environment with common modeling libraries. It is intended to be used by data scientists to train new candidate models entity resolution and / or job clustering.
```

Comment:
this is the place where I think SageMaker fits in very nicely. Again, I am not saying we should use this (it may be more expensive than what you are suggesting). But, it is super easy to work with sagemaker.

Reply:
Sure, maybe. The easiest way to prototype is in Jupyter, IMHO. Let's not confuse implementation for service boundaries. It doesn't have to stay Jupyter forever